test/controllers/content_items_controller_test.rb

```diff
@@ -150,7 +152,21 @@ class ContentItemsControllerTest < ActionController::TestCase
     assert_equal content_item['links']['ordered_related_items'], assigns[:content_item].content_item['links']['ordered_related_items']
   end
 
+  test "gets item from content store and replaces ordered_related_items when feature flag header is specified but recommended links turned off" do
+    HttpFeatureFlags.instance.add_http_feature_flag(FeatureFlagNames.recommended_related_links, 'false')
+    request.headers[FeatureFlagNames.recommended_related_links] = 'true'
+
+    content_item = content_store_has_schema_example('case_study', 'case_study')
+
+    get :show, params: { path: path_for(content_item) }
+    assert_response :success
+    assert_empty content_item['links']['ordered_related_items'], 'Content item should have existing related links'
+    refute_empty content_item['links']['suggested_ordered_related_items'], 'Content item should have existing suggested related links'
+    assert_equal [], assigns[:content_item].content_item['links']['ordered_related_items']
```

Comment:
The test name says it replaces ordered_related_items but this is asserting that nothing is assigned to the ordered_related_items, isn't that backwards?